main/src/main/java/cgeo/geocaching/ui/SimpleItemListView.java

```diff
@@ -271,6 +274,33 @@ private void handleClick(final int adapterPos) {
         public void onBindViewHolder(@NonNull final ItemListViewHolder holder, final int position) {
             holder.fillData(getItem(position));
 
+            if (!model.getDisabledItems().isEmpty()) {
```

Comment:
This might probably not work when a list first has disabled items, then later all disabled items are removed (programatically). Not entirely sure myself though.

Reply:
Did you test this?